src/app/js/formats/hierarchy/Hierarchy.js

```diff
@@ -685,6 +698,16 @@ class Hierarchy extends PureComponent {
         this.update();
     }
 
+    handleMouseEnter = () => {
+        this.isHovered = true;
+        alert('enter');
+    };
+
+    handleMouseLeave = () => {
+        this.isHovered = true;
+        alert('leave');
+    };
+
```

Comment:
For the moment, `handleMouseEnter` and `handleMouseLeave` are not used.
If you don't plan to use them, remove them.

Reply:
Also, eslint(react/prop-types) complains about lines 665 (for `maxLabelLength`), line 368 (`labelOffset`), line 204 (`params.minimumScaleValue`), line 183 (`colors`), and line 113 (`formatData`): missing in props validation.

And there is still a problem line 409, with an unused parameter.

Reply:
Saw it yesterday and currently fixing the eslint indications !